src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -166,6 +166,15 @@ pip3 install -r requirements.txt
 $ cwltool --version
 /usr/local/bin/cwltool ${this.cwltoolVersion}
 \`\`\`
+
+#### Part 5 - Install Nextflow (Optional)
+The Dockstore CLI does not run Nextflow workflows. Users must run them directly by using the Nextflow command line tool.
```

Comment:
must -> can 
(minor, just opening the door that there are other options like Nextflow Tower as well)